site/Calendar.php

```diff
@@ -1,29 +1,7 @@
 <html>
-	<head>
-	<link rel="stylesheet" type="text/css" href="Style.css">
-	<link href="https://fonts.googleapis.com/css?family=Raleway"
-	rel="stylesheet">
-		<title>
-			St. Louis Theatre Calendar
-		</title>
-	</head>
-	<body>
-		<div class="Header">
-			<?php include 'include/header.php';
-				PrintHeader();
-			?>
-		</div>
-		<div class="PageTitle">
-			<h2>
-				Calendar
-			</h2>
-		</div>
-		<div class="DivideLine">
-		</div>
-		<div class="IncludeFooter">
-			<?php include 'include/footer.php';
-				PrintFooter();
-			?>
-		</div>
-	</body>
-</html>
+	<?php
+		include_once('init.php');
+		$SiteTitle = 'St. Louis Theatre Calendar';
+		$PageTitle = 'Calendar';
+		PrintHeader($SiteTitle, $PageTitle);
```

Comment:
This isn't a problem, but I just want to note that you can pass strings directly into functions without saving them as a variable first. So you could replace lines 4-6 with:
```
PrintHeader('St. Louis Theatre Calendar', 'Calendar');
```